docs/docs/use-cases/keda.md

```diff
@@ -0,0 +1,201 @@
+---
+comments: true
+---
+
+# Scaling Workloads with KEDA based on Keptn metrics
+
+If you want to use [KEDA](https://keda.sh/) to scale your workloads, you can set it up
+to consume metrics from Keptn.
+This gives you great flexibility for your setup since Keptn can
+consolidate different observability providers for you and KEDA
+will simplify the scaling operations for you.
+
+This use case is enabled by the Keptn Metrics
+[custom API](https://kubernetes.io/docs/reference/external-api/custom-metrics.v1beta2/)
+that the Keptn Metrics Operator provides.
+
+## Before you begin
+
+For this use case presentation, a few components need to be deployed to the
+cluster in order to have a full setup:
+
+- Keptn Metrics Operator: Will be used for metrics consolidation.
+
+    For more information about installation please refer to the
+    [installation guide](../installation/index.md).
+
+    > The Keptn Lifecycle Operator does not need to be installed for this use-case.
+
+- [KEDA](https://keda.sh/): Will be used for scaling.
+- [Prometheus](https://prometheus.io/): Will be used as metrics provider.
+  
+    For more information about how to install Prometheus into your cluster, please
+    refer to the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/installation/).
+
+## Deploy sample application
+
+First, we need to deploy our application to the cluster.
+For this we are going to
+use a single service `podtato-head` application.
+
+<!-- markdownlint-disable MD046 -->
+
+=== "deployment.yaml"
+
+    ```yaml
+    {% include "./assets/keda/sample-app.yaml" %}
+    ```
+
+=== "service.yaml"
+
+    ```yaml
+    {% include "./assets/keda/sample-service.yaml" %}
+    ```
+
+<!-- markdownlint-enable MD046 -->
+
+Please create a `podtato-kubectl` namespace and apply the above manifests
+to your cluster and continue with the next steps.
+After applying, please make sure that the application is up and running:
+
+```shell
+$ kubectl get pods -n podtato-kubectl
+podtato-head-entry-58d6485d9b-ld9x2         1/1     Running     (2m ago)
+```
+
+## Create KeptnMetric and KeptnMetricsProvider custom resources
+
+To be able to react on the metrics of our application, we need to create
+`KeptnMetrics` and `KeptnMetricsProvider` custom resources.
```

Comment:
It might be nice to have these link to the CRD ref pages if they want to look up the syntax while looking at the examples.

Reply:
i will link the first occurrence